examples/sendListMessage/main.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"encoding/gob"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/Rhymen/go-whatsapp/binary/proto"
+
+	qrcodeTerminal "github.com/Baozisoftware/qrcode-terminal-go"
+	"github.com/Rhymen/go-whatsapp"
+)
+
+func main() {
+	//create new WhatsApp connection
+	wac, err := whatsapp.NewConn(5 * time.Second)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error creating connection: %v\n", err)
+		return
+	}
+
+	err = login(wac)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error logging in: %v\n", err)
+		return
+	}
+
+	<-time.After(6 * time.Second)
+
+	previousMessage := "How about u"
+	quotedMessage := proto.Message{
+		Conversation: &previousMessage,
+	}
+
+	ContextInfo := whatsapp.ContextInfo{
+		QuotedMessage:   &quotedMessage,
+		QuotedMessageID: "F229B6BCCE2A0D6BBB0AA56110887D20",
+		Participant:     "919315346424@s.whatsapp.net", //Who sent the original message
+	}
+
+	title := "List 1"
+	description := "description 1"
+	rowId := "rowId1"
+
+	var Section = []*proto.Section{ ///
```

Comment:
There could be a better & easy way to do this but i went with this as i copied from my code.